main.go

```diff
@@ -13,6 +13,8 @@ import (
 var Router = mux.NewRouter().StrictSlash(true)
 
 func main() {
+	Router.PathPrefix("/static/").Handler(http.StripPrefix("/static/", http.FileServer(http.Dir("./static/"))))
```

Comment:
@pashagap, let's move it to one of `flowAuthCode*.go` files cause this is related only to "authentication code" flow

Reply:
done